pkg/internal/common/logging/metadata/opts.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
How do you intend to expose the functions in a later PR?

Reply:
I just use metadata.GetLoggerOpts() 
I am going to add one more package under internal `internal/common/logging/modlog` which will use this package.